client/state/sharing/publicize/publicize-actions/reducer.js

```diff
@@ -97,11 +106,23 @@ export const editingSharePostAction = createReducer( {}, {
 		( state, { siteId, postId, actionId } ) => updateDataForPost( true, state, siteId, postId, actionId ),
 } );
 
+export const schedulingSharePostActionStatus = createReducer( {}, {
```

Comment:
Yes, I agree that naming of this function is unfortunate and would benefit from renaming.
This however, is definitely out of the scope of this PR.